django_spanner/__init__.py

```diff
@@ -61,7 +62,12 @@ def gen_rand_int64():
 def autofield_init(self, *args, **kwargs):
     kwargs["blank"] = True
     Field.__init__(self, *args, **kwargs)
-    self.default = gen_rand_int64
+
+    if (
+        django.db.connection.settings_dict["ENGINE"] == "django_spanner"
+        and self.default == NOT_PROVIDED
+    ):
```

Comment:
Using the INT64 random key only in case it's a Spanner database and default value/function for the auto-field is not provided. Thus, outside Spanner database it'll work as always, and for Spanner users will still be able to override the key with whatever they want.